Source/WTF/wtf/FastMalloc.h

```diff
@@ -338,6 +338,10 @@ using WTF::fastAlignedFree;
         ASSERT(location); \
         return location; \
     } \
+    static void deleteAfterDestruction(void* p) \
```

Comment:
Should we call this freeAfterDestruction? Delete sorta implies we destroy and free.

Reply:
I guess “AfterDestruction” implies it’s after the  destructor runs. But I still think I’d call this freeAfterDestruction

Reply:
OK, changed it to `freeAfterDestruction`.